packages/core-data/src/actions.js

```diff
@@ -537,6 +537,25 @@ export const saveEntityRecord =
 				'wp_registered_template',
 				recordId
 			);
+			// Evaluate optimized edits.
+			// (Function edits that should be evaluated on save to avoid expensive computations on every edit.)
+			for ( const [ key, value ] of Object.entries( record ) ) {
+				if ( typeof value === 'function' ) {
+					const evaluatedValue = value(
+						select.getEditedEntityRecord( kind, name, recordId )
```

Comment:
Honestly not sure why this works because this is basically:
```
wp.data.select('core').getEditedEntityRecord('postType','wp_template','twentytwentyfour//home');
```

And there should be no edited entities with those IDs 🤔

Reply:
Omg, you're right!